mediapipe/examples/android/src/java/com/google/mediapipe/apps/instantmotiontracking/Sticker.java

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.mediapipe.apps.instantmotiontracking;
+
+import java.util.ArrayList;
+
+// Stickers represent a unique object to render and manipulate in an AR scene.
+// A sticker has a sticker_id (a unique integer identifying a sticker object to
+// render), x and y normalized anchor coordinates [0.0-1.0], user inputs for
+// rotation in radians, scaling, and a renderID (another unique integer which
+// determines what object model to render for this unique sticker).
+
+public class Sticker {
+
+  private int renderID = 0; // ID of object to render
+  private float xAnchor, yAnchor; // Normalized X and Y coordinates of anchor
```

Comment:
To clarify, let's say where the (0,0) corner lies for our normalized x,y coordinates (top-left, bottom-right, etc.)